modules/search/github.py

```diff
@@ -70,18 +79,22 @@ def module_api(self):
 	bactery = ['marketplace', 'pulls', 'explore', 'issues',\
 				'notifications', 'account', 'settings', 'login',\
 				'about', 'pricing', 'site']
-	for user in set(explore['Github']):
+	for user in set(explore['Github'][1:]):
 		user = user.split('/')[1]
 		if user not in bactery:
 			output['usernames'].append(user)
+	for user in USERS[1:]:
+		user = user.split('/')[1]
+		output['usernames'].append(user)
 
 	for link in self.reglib().filter(r"https://(www\.)?github\.com/[\w-]{1,39}/[\w\-\.]+", list(set(LINKS))):
 		repo = re.search(r"(https://(www\.)?github\.com/([\w-]{1,39})/[\w\-\.]+)", link)
 		if repo.group(3) not in bactery:
 			repo = repo.group(0)
 			if repo not in output['repositories']:
 				output['repositories'].append(repo)
-
+	for link in REPO:
```

Comment:
why for?

Reply:
REPO is list of repos coming from github api. It's not scraping from webpages but send repositories based on the keyword query directly. And yes,  no json formatting was needed. List of matched strings according to match-header

Reply:
any problem with the for loop? I saw usernames getting appended in the same way. Both are lists